### DISCUSSION TITLE
İbax referans coinleri gelmeme sorunu

### DISCUSSION THREAD

**Discussion by Theyavo** [IBAX-io/go-ibax#1473] [Soru Cevap - Türkçe]
İbax için 5 gün önceye kadar referans kasıyordum coinler geliyordu ama şimdi kaç kişiye yaptırırsam yaptırayım gelmiyor

**Comment by Yusufknk** [comment]
Her iki tarafın görevleri eksiksiz olarak yapması aynı zamanda sosyal medya kanallarını takipten çıkarmamaları gerekir. Bu koşullar sağlandığı takdirde en az 24 saat bekleyin.
